tests/b2share_unit_tests/records/test_records_index.py

```diff
@@ -33,29 +33,35 @@
 from invenio_indexer.api import RecordIndexer
 from invenio_records.api import Record
 from b2share.modules.deposit.api import Deposit
-
+from time import sleep
```

Comment:
flush method was not sufficient to refresh the indices

use wait_for_completion instead